integration_tests/dbt_utils/dbt_project.yml

```
@@ -17,46 +17,26 @@ clean-targets:         # directories to be removed by `dbt clean`
     - "dbt_modules"
     
 vars:
-  dbt_utils_dispatch_list: ['tsql_utils']
+  dbt_utils_dispatch_list: ['tsql_utils', 'dbt_utils']
```

added this as a hail mary.... so close now!
```
almost there... do I have to add a var to `tsql-utils/integration_tests/dbt_utils/packages.yml`?
```
  In dispatch: No macro named 'limit_zero' found
      Searched for: 'dbt_utils.sqlserver__limit_zero', 'dbt_utils.default__limit_zero'
```
```

---

Good call, this should actually be the `name` of the project (`integration_tests/dbt_utils/dbt_project.yml`) that defines `sqlserver__limit_zero` (`integration_tests/dbt_utils/macros/limit_zero.yml`):
```suggestion
  dbt_utils_dispatch_list: ['tsql_utils', 'tsql_utils_dbt_utils_integration_tests']
```